sdk/loadtestservice/load-testing-rest/src/beginTestRunStatus.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortController, AbortSignalLike } from "@azure/abort-controller";
+import { CancelOnProgress, OperationState, SimplePollerLike } from "@azure/core-lro";
+import { TestRunStatusPoller, PolledOperationOptions } from "./models";
+import { AzureLoadTestingClient } from "./index.js";
+import { TestRunGet200Response } from "./responses";
+import { isUnexpected } from "./isUnexpected";
+
+/**
+ * Creates a poller to poll for test run status.
+ * @param client - The Load Testing client.
+ * @param options - The operation options.
+ * @returns A poller which can be called to poll until completion of the job.
+ */
+export async function beginTestRunStatus(
+  client: AzureLoadTestingClient,
+  testRunId: string,
```

Comment:
Is there any strong reason to pass the client and the ID instead of passing the response instead?